packages/nc-gui/components/smartsheet/Grid.vue

```diff
@@ -208,48 +201,44 @@ const { selectCell, selectBlock, selectedRange, clearRangeRows, startSelectRange
     if (cmdOrCtrl) {
       switch (e.key) {
         case 'ArrowUp':
-          selected.row = 0
-          selected.col = selected.col ?? 0
+          selectedCell.row = 0
+          selectedCell.col = selectedCell.col ?? 0
           scrollToCell?.()
           editEnabled = false
           return true
         case 'ArrowDown':
-          selected.row = data.value.length - 1
-          selected.col = selected.col ?? 0
+          selectedCell.row = data.value.length - 1
+          selectedCell.col = selectedCell.col ?? 0
           scrollToCell?.()
           editEnabled = false
           return true
         case 'ArrowRight':
-          selected.row = selected.row ?? 0
-          selected.col = fields.value?.length - 1
+          selectedCell.row = selectedCell.row ?? 0
+          selectedCell.col = fields.value?.length - 1
           scrollToCell?.()
           editEnabled = false
           return true
         case 'ArrowLeft':
-          selected.row = selected.row ?? 0
-          selected.col = 0
+          selectedCell.row = selectedCell.row ?? 0
+          selectedCell.col = 0
           scrollToCell?.()
           editEnabled = false
           return true
       }
     }
-  },
-)
+  })
 
 function scrollToCell(row?: number | null, col?: number | null) {
-  row = row ?? selected.row
-  col = col ?? selected.col
+  row = row ?? selectedCell.row
+  col = col ?? selectedCell.col
   if (row !== undefined && col !== undefined && row !== null && col !== null) {
     // get active cell
     const rows = tbodyEl.value?.querySelectorAll('tr')
     const cols = rows?.[row].querySelectorAll('td')
     const td = cols?.[col === 0 ? 0 : col + 1]
-
     if (!td || !gridWrapper.value) return
-
     const { height: headerHeight } = tableHead.value!.getBoundingClientRect()
     const tdScroll = getContainerScrollForElement(td, gridWrapper.value, { top: headerHeight, bottom: 9, right: 9 })
-
     if (rows && row === rows.length - 2) {
```

Comment:
Please bring back the spacing(newline) between these statements, that way it looks cleaner.